packages/core/src/cli.ts

```diff
@@ -211,5 +211,11 @@ async function boostrapCodegen(location: string, watch: boolean) {
       },
     },
   })
-  return generate(ctx, true)
+
+  await generate(ctx, true)
+
+  await appendFile(
+    baseDirectory + '/fuse' + '/index.ts',
+    '\nexport * from "fuse/client"',
+  )
```

Comment:
This might interfere with the change made in #100 since the hooks are run as part of the codegen